File: Heat/Models/Enum/ThermalUnitKindEnum.vb
Imports System.ComponentModel.DataAnnotations

Public Enum ThermalUnitKindEnum
    <Display(name:="Gruppo termico singolo")> _
    SingleThermalUnit = 1

    <Display(name:="Tubo/nastro radiante")> _
    TubeOrRadiantStrip = 2

    <Display(name:="Gruppo termico modulare")> _
    ModularThermalUnit = 3

    <Display(name:="Generatore di aria calda")> _
        HotAirGenerator = 4

End Enum

File: Heat.ConvertedToC#/Models/Enum/ThermalUnitKindEnum.cs
using System.ComponentModel.DataAnnotations;
namespace Heat
{

    public enum ThermalUnitKindEnum
	{
		[Display(Name = "Gruppo termico singolo")]
		SingleThermalUnit = 1,

		[Display(Name = "Tubo/nastro radiante")]
		TubeOrRadiantStrip = 2,

		[Display(Name = "Gruppo termico modulare")]
		ModularThermalUnit = 3,

		[Display(Name = "Generatore di aria calda")]
		HotAirGenerator = 4

	}
}
